src/config.rs

```diff
@@ -0,0 +1,34 @@
+use std::{borrow::Cow, path::PathBuf};
+
+use figment::{
+    providers::{Env, Format, Yaml},
+    Figment,
+};
+
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct Config {
```

Comment:
Should this be `SentrySDKConfig`?

Reply:
Nah this is the global config. I'll add more to it dw

Reply:
have another PR after this to add the kafka clusters config